cluster/ci/monitoring/mixins/grafana_dashboards/prow.jsonnet

```diff
@@ -14,6 +14,27 @@ local dashboardConfig = {
         uid: '970b051d3adfd62eb592154c5ce80377',
     };
 
+local yaxesConfig = {
+ "yaxes": [
+        {
+          "format": "decbytes",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        },
+        {
+          "format": "decbytes",
```

Comment:
format for Y has been parameterized in jsonnet lib. Instead of this, we could do
`graphPanel.new( ..., formatY1=`decbytes`)` in below when you create the `graphPanel`.

Reply:
niceeeee... addressed.